src/applications/personalization/profile/hooks/useNotificationSettingsUtils.js

```diff
@@ -262,13 +263,22 @@ export const useNotificationSettingsUtils = () => {
       [itemIds],
     );
 
+  const usePaperlessDeliveryGroup = () => {
+    return getAvailableGroups(communicationPreferences, [3]).filter(
```

Comment:
I guess `channelsWithContactInfo` still works here. I'm not sure what it should be called. 
`function getAvailableGroups(communicationPreferences, channelsWithContactInfo) {`

Reply:
Our Paperless Delivery group has a channel of `3`. It looks like they use channel `1` for texts and `2` for emails. I'm not sure why we were given a `3`. I'll bring it up in the next meeting with the VA Profile team.